homeassistant/components/rtorrent/sensor.py

```diff
@@ -102,6 +114,11 @@ def update(self):
         multicall = xmlrpc.client.MultiCall(self.client)
         multicall.throttle.global_up.rate()
         multicall.throttle.global_down.rate()
+        multicall.d.multicall2("", "main")
```

Comment:
What is multicall2 doing?

Reply:
https://rtorrent-docs.readthedocs.io/en/latest/cmd-ref.html

"d.multicall2 iterates over all items in view and calls the given commands on each, assembling the results of those calls in a row per item. Typically, the given commands either just have a side effect (e.g. d.stop), or return some item attribute (e.g. d.name)."

Reply:
Okay thanks for the explanation 👍 Does the first paramater have to be an empty string or could it be `None` as well?

Reply:
As far as I know it must be an empty string in this case.

Reply:
Could you test it with None?

Reply:
Sure I thought about that. I will test it on this weekend.

Reply:
Getting this error message:
"cannot marshal None unless allow_none is enabled" 
I assume it has to be an empty string.

Reply:
Okay thanks for testing.